test/suites/migration.sh

```diff
@@ -21,6 +21,15 @@ test_migration() {
   lxc_remote move l1:nonlive l2:
   lxc_remote config show l2:nonlive/snap0 | grep user.tester | grep foo
 
+  # test `lxd import`
+  if [ "${LXD_BACKEND}" = "zfs" ]; then
+    lxc_remote init testimage backup
+    lxc_remote snapshot backup
+    sqlite3 "${LXD_DIR}/lxd.db" "DELETE FROM containers WHERE name='backup'"
```

Comment:
Isn't that effectively the same as this? The idea is you have a valid zfs pool, but no database.